applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/usteer.js

```diff
@@ -0,0 +1,472 @@
+								hostl
+							]);
+		}
+		for(var wlan in Remoteinfo) {
+			var hostl=''
+			for(var mac in Clients) {
+				if (typeof Clients[mac] !== 'undefined') 
+					if (typeof Clients[mac][wlan] !== 'undefined') 
+						if (String(Clients[mac][wlan]['connected']).valueOf()==String("true").valueOf()) {
+							var foundname=mac;
+							var macUp=String(mac).toUpperCase()
+							if (typeof Hosts[macUp] !== 'undefined') {
+								if ((String(Hosts[macUp]['ipaddrs'][0]).length>0) && (typeof Hosts[macUp]['ipaddrs'][0] !== 'undefined')) {
+									foundname=Hosts[macUp]['ipaddrs'][0]; 
+								}
+								if ((String(Hosts[macUp]['name']).length>0) &&	(typeof Hosts[macUp]['name'] !== 'undefined')) {
+									foundname=Hosts[macUp]['name'];
+								}
+							}
+							hostl += '%h\u2003'.format(foundname);
+				}
+			}
+			compactconnectioninfo_table_entries.push([
+								'<nobr>'+wlan+'</nobr>', 
+								Remoteinfo[wlan]['ssid'],
+								Remoteinfo[wlan]['freq'],
+								Remoteinfo[wlan]['load'],
+								Remoteinfo[wlan]['n_assoc'],
+								hostl
+			]);
+		}
+		cbi_update_table(compactconnectioninfo_table, compactconnectioninfo_table_entries, E('em', _('No data')));
+		body.appendChild(compactconnectioninfo_table);
+		return E('div', { 'class': 'cbi-section cbi-tblsection' }, [body]);
+	}
+});
+
+var mapdata = { status: {}, hearingmap: {},  settings: {} };
+
+return view.extend({
+	callHostHints: rpc.declare({
+		object: 'luci-rpc',
+		method: 'getHostHints',
+		expect: { '': {} }
+	}),
+	callGetRemotehosts: rpc.declare({
+		object: 'usteer',
+		method: 'remote_hosts',
+		expect: {  '': {}}
+	}),
+	callGetRemoteinfo: rpc.declare({
+		object: 'usteer',
+		method: 'remote_info',
+		expect: { '': {} }
+	}),
+	callGetLocalinfo: rpc.declare({
+		object: 'usteer',
+		method: 'local_info',
+		expect: { '': {} }
+	}),
+	callGetClients: rpc.declare({
+		object: 'usteer',
+		method: 'get_clients',
+		expect: { '': {} }
+	}),
+	load: function() {
+		return Promise.all([
+			this.callHostHints(),
+			this.callGetRemotehosts(),
+			this.callGetRemoteinfo(),
+			this.callGetLocalinfo(),
+			this.callGetClients()
+		]);
+	},
+	render: function(data) {
+		var m, s, o;
+		Hosts = data[0];
+		Remotehosts=data[1];
+		Remoteinfo=data[2];
+		Localinfo=data[3];
+		Clients=data[4];
+
+		m = new form.JSONMap(mapdata, _('Usteer'));		
+		m.tabbed = true;
+
+		s = m.section(form.NamedSection, 'status', 'Status');
+		s.anonymous = true;
+
+		o = s.option(Clientinfooverview);
+		
+		s = m.section(form.NamedSection, 'hearingmap', 'Hearing map');
+		s.anonymous = true;
+
+		o = s.option(HearingMap);
+
+		s = m.section(form.TypedSection, 'settings','Settings',  _('First four options are mandatory. Also be sure to enable rrm reports, 80211kv, etc see: https://openwrt.org/docs/guide-user/network/wifi/usteer'));
+		s.anonymous = true;
+
+		s.option(form.Value, 'network', _('Network'), _('The network interface for inter-AP communication)'));
```

Comment:
One ) too much?
 But it is inside a string, so it shouldn't hurt, right?

Reply:
Let me try, but I dont think an extra ) inside a string can be the issue. I think it is related that i switched to` m = new form.JSONMap(mapdata, 'Usteer'); ` instead of `form.Map`, i think that changed the way the uci settings are mapped. If i dont then the different tabs do not work correctly. But I am not sure how to fix it.

Reply:
luci-app-nlbwmon might offer you an example of pretty similar structure. Three pages shown as tabs, one item in menu.

Reply:
I think its fixed. I moved back to a form.Map and tabbed the section instead of the map.  Please have another look